frontend/src/pages/talks/TalkEntryList.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box } from '@mui/material';
+import React from 'react';
+
+import TalkSingleEntry from 'src/pages/talks/TalkSingleEntry';
+import { TalkEntry } from 'src/services/mocks';
+
+const TalkEntryList = ({ talks }: { talks: Array<TalkEntry> }) => {
+  return (
+    <>
+      {talks.map((talk) => (
+        <Box key={talk.uid} mx={1} my={2}>
+          <TalkSingleEntry key={`q_${talk.title}`} talk={talk} />
```

Comment:
Let's use the `key={talk.name}` here (see my comment in the JSON file.

Reply:
Good idea now that we have this attribute, it's changed